nilearn/plotting/img_plotting.py

```diff
@@ -640,7 +640,10 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
         nilearn.plotting.plot_prob_atlas : To simply plot probabilistic atlases
             (4D images)
 
+        Notes
+        -----
 
+        A small threshold is applied by default to eliminate numerical background noise. 
```

Comment:
Same here line too long (88 > 79 characters).

Could you break the sentence into two lines ? and
It would be better if you could drag Notes section above See Also.
Thanks